go/client/cmd_chat_api.go

```diff
@@ -239,7 +223,7 @@ func (c *CmdChatAPI) ReadV1(ctx context.Context, opts readOptionsV1) Reply {
 	if opts.ConversationID == 0 {
 		// resolve conversation id
 		query := c.getInboxLocalQuery(opts.ConversationID, opts.Channel)
-		gilres, err := client.GetInboxLocal(ctx, chat1.GetInboxLocalArg{
+		gilres, err := client.GetInboxAndUnboxLocal(ctx, chat1.GetInboxAndUnboxLocalArg{
```

Comment:
this one doesn't need to unbox, right?  because to read the conversation, the thread will be unboxed.